pkg/apiserver/registry/controlplane/egressgroup/rest.go

```diff
@@ -54,6 +54,9 @@ func (r *REST) New() runtime.Object {
 	return &controlplane.EgressGroup{}
 }
 
+func (r *REST) Destroy() {
```

Comment:
@tnqn do we have any need for `Destroy` in any of our stores at the moment?

Reply:
I assume this will be executed when apiserver performs graceful shutdown?
I don't think of anything we must do for the stores at the moment as there are ephermeral data anyway.